sdlf-cicd/lambda/domain-cicd/src/lambda_function.py

```diff
@@ -182,15 +182,16 @@ def create_team_repository_cicd_stack(domain, team_name, template_body_url, clou
     }
     json_data = json.dumps(data).encode('utf-8')
     req = Request(url, data=json_data, headers=headers, method='POST')
-    # Send the request and handle the response
+
+    # TODO: Check repo exists already and only send create command
     try:
         with urlopen(req) as response:
             response_body = response.read().decode('utf-8')
-            print(response_body)
+            logger.error(response_body)
```

Comment:
In this case you would use `logger.info()` - you're in the `try` not the `except`.

Reply:
fixed